better_user_interface.py

```diff
@@ -51,12 +50,9 @@ def create_screen(update_func):
     button = customtkinter.CTkButton(root, text="Run", command=update_func)
     button.grid(row=6, columnspan=2, pady=10)
 
+    global textbox
     textbox = customtkinter.CTkTextbox(root)
     textbox.insert("0.0", answer_form_gpt)  # insert at line 0 character 0
```

Comment:
Forgot something here.
Instead of:
```
textbox.insert("0.0", answer_form_gpt)
```
It should be:
```
textbox.insert("0.0", "Answer")
```
... or whatever should be displayed by default